website/docs/components/page-header/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,237 @@
+## Usage
+
+### When to use
+
+- To display the title or topmost heading within a page.
+- To communicate metadata pertaining to the page; e.g., status, description, subtitle, etc.
+- To highlight page-level actions and functions.
+
+### When not to use
+
+- To communicate page-level information anywhere other than the top of the page.
+
+## Title
+
+Displays the title or topmost heading of the page. This should be limited to no more than four words and not exceed 50 characters.
```

Comment:
Hi Jory, I think the "no more than four words" is hard to follow because there are page titles that are showing user-defined values. For example, the HCP Organization name, which is user-defined and is used on the organization dashboard page as the title. There is a character number limitation of 36, but no limitation on how many words it can contain (and it allows spaces). 
For the "50 characters" limitation, so far it seems fine to me - I know Terraform Cloud's org name has a 40 characters limitation, and most of the user-defined names on HCP have a 36 characters limitation. But my research is not exhaustive so there is a chance that the 50 limitation is not applicable. But I agree that this is a good rule of thumb in most of the scenarios.

Reply:
I forgot to mention that the use case of having the org name be the page title is an ongoing ticket to be implemented here once the page header component is available: https://hashicorp.atlassian.net/browse/HCPF-639

Reply:
Ok for now I'll leave the recommendation on the character limit, but remove the note on number of words. That's a good point on user generated titles.